.travis.yml

```diff
@@ -1,10 +1,12 @@
 before_install:
+  - gem install bundler
   - gem update bundler
```

Comment:
`gem install bundler` should be enough, and it should always be used (it's not a fix specific to travis and jruby-head), it's just that maybe because of travis relying on RVM we get bundler when installing MRI, but with a standard MRI install you wouldn't have bundler.